Cython/Includes/cpython/datetime.pxd

```diff
@@ -119,21 +193,41 @@ cdef extern from "datetime.h":
     int PyDateTime_DATE_GET_MINUTE(object o)
     int PyDateTime_DATE_GET_SECOND(object o)
     int PyDateTime_DATE_GET_MICROSECOND(object o)
+    int PyDateTime_DATE_GET_FOLD(object o)
+    object PyDateTime_DATE_GET_TZINFO(object o)
 
     # Getters for time (C macros).
     int PyDateTime_TIME_GET_HOUR(object o)
     int PyDateTime_TIME_GET_MINUTE(object o)
     int PyDateTime_TIME_GET_SECOND(object o)
     int PyDateTime_TIME_GET_MICROSECOND(object o)
+    int PyDateTime_TIME_GET_FOLD(object o)
+    object PyDateTime_TIME_GET_TZINFO(object o)
 
     # Getters for timedelta (C macros).
     int PyDateTime_DELTA_GET_DAYS(object o)
     int PyDateTime_DELTA_GET_SECONDS(object o)
     int PyDateTime_DELTA_GET_MICROSECONDS(object o)
 
+    # Constructors
+    object PyDateTime_FromDateAndTimeAndFold(
+        int year, int month, int day, int hour, int min, int sec, int usec, int fold
+    )
+    object PyTime_FromTimeAndFold(int hour, int minute, int second, int usecond, int fold)
+
+    # We don't need this first one
+    # object PyTimeZone_FromOffset(int offset)
+    object PyTimeZone_FromOffsetAndName(int offset, str name)
+
+    # Constructors for the DB API
+    object PyDateTime_FromTimeStamp(object args)
+    object PyDate_FromTimeStamp(object args)
+
```

Comment:
This was the only way from what I found that we could support these new features while being able to work on previous version. Interesting enough, I am getting some warning while compiling though:
```
testy.c(2806): warning C4047: 'function': 'PyObject *' differs in levels of indirection from 'int'
testy.c(2806): warning C4024: 'function through pointer': different types for formal and actual parameter 1
testy.c(2884): warning C4013: 'PyDateTime_FromTimeStamp' undefined; assuming extern returning int
testy.c(2884): warning C4047: '=': 'PyObject *' differs in levels of indirection from 'int'
testy.c(2945): warning C4013: 'PyDate_FromTimeStamp' undefined; assuming extern returning int
testy.c(2945): warning C4047: '=': 'PyObject *' differs in levels of indirection from 'int'
```

Not sure how to satisfy the compiler there, `PyDateTime_FromTimeStamp` and `PyDate_FromTimeStamp` should not be much different from the other functions.

(I am gonna install an older version of Python and get to the issue wth `PyDateTime_TimeZone_UTC?` along with fully testing this)